tests/templates/kuttl/mount-dags-gitsync/03-install-airflow-cluster.yaml.j2

```diff
@@ -35,46 +35,32 @@ spec:
   loadExamples: false
   exposeConfig: false
   credentialsSecret: test-airflow-credentials
-  volumes:
-    - name: external-dags
-      persistentVolumeClaim:
-        claimName: test-pvc-airflow
-  volumeMounts:
-    - name: external-dags
-      mountPath: /stackable/external-dags
+  clusterConfig:
+    dagsGitSync:
+      - name: git-sync
+        repo: https://github.com/stackabletech/airflow-operator
+        branch: "feat/git-sync" # TODO update to main before merging
```

Comment:
Highlighting comment which might be forgotten.

Reply:
Done.